src/accountsdb/db.zig

```diff
@@ -2011,6 +2056,8 @@ pub const AccountsDB = struct {
         );
     }
 
+    /// index and validate an account file.
+    /// NOTE: should only be called in tests/benchmarks
```

Comment:
What do you think about adding an assertion?

Reply:
i dont think we have a way to tell if code is in a benchmark rn?

Reply:
A simple (hacky) way to tell that would be to `comptime assert(@import("root") == @import("../benchmarks.zig"));`.

Reply:
gonna leave it for now with an open ticket: https://github.com/orgs/Syndica/projects/2/views/1?pane=issue&itemId=75090925